examples/ads.amp.html

```diff
@@ -1536,6 +1537,16 @@ <h2>mox</h2>
       data-h="405">
   </amp-ad>
 
+  <h2>My6sense</h2>
```

Comment:
I did not see an example creative load here, would it be possible to return a sample ad? It helps us catch regressions etc.

Reply:
Hi @calebcordry, when i'm running this file locally i'm getting our sample ad,
Can you pls tell me if you are getting widget3.js in network tab, if so can you share it with me, tnx!

![1](https://user-images.githubusercontent.com/16783303/77995517-9a279380-7334-11ea-950f-f8916d13fa01.PNG)